WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -66,15 +67,46 @@
             <% else %>
               <%= alert :success, t('registrations.accepted') %>
             <% end %>
+            <div>
+              <% if @competition.connected_stripe_account_id %>
+                <% if @competition.connected_stripe_account_id && @registration.outstanding_entry_fees > 0 %>
+                  <% if @registration.paid_entry_fees > 0 %>
+                    <%= alert :info do %>
```

Comment:
Not something we have to do as part of this PR, but we're definitely going to want to translate all this later on. Could you either do it now, or make an issue for it so we can get it later?